PINCache/PINDiskCache.m

```diff
@@ -304,17 +317,19 @@ + (BOOL)moveItemAtURLToTrash:(NSURL *)itemURL
 
 + (void)emptyTrash
 {
-    dispatch_async([self sharedTrashQueue], ^{
-        NSError *searchTrashedItemsError = nil;
-        NSArray *trashedItems = [[NSFileManager defaultManager] contentsOfDirectoryAtURL:[self sharedTrashURL]
-                                                              includingPropertiesForKeys:nil
-                                                                                 options:0
-                                                                                   error:&searchTrashedItemsError];
-        PINDiskCacheError(searchTrashedItemsError);
+    dispatch_async([PINDiskCache sharedTrashQueue], ^{
+        NSURL *trashURL;
+        
+        [[PINDiskCache sharedLock] lock];
```

Comment:
This is awesome! Do you mind adding a comment explaining this?

Something along the lines of:
If _sharedTrashURL is unset, there's nothing left to do because it hasn't been accessed and therefore items haven't been added to it. If it is set, we can just remove it.

We also need to nil out _sharedTrashURL so that a new one will be created if there's an attempt to move a new file to the trash.